setup.py

```diff
@@ -25,7 +25,9 @@
     "jinja2",
     "sqlglot>=11.3.7",
     "ploomber-core>=0.2.4",
-    'importlib-metadata;python_version<"3.8"'
+    'importlib-metadata;python_version<"3.8"',
+    "dockerctx",
```

Comment:
install_requires are mandatory requirements for end-users; and we don't want docker/dockerctx there since those are only required for testing. they should go in the development requirements section